render/detailed/links.go

```diff
@@ -0,0 +1,169 @@
+package detailed
+
+import (
+	"bytes"
+	"net/url"
+	"strings"
+	"text/template"
+
+	"github.com/weaveworks/scope/probe/docker"
+	"github.com/weaveworks/scope/report"
+
+	"github.com/ugorji/go/codec"
+)
+
+// MetricLink describes a link referencing a metric.
```

Comment:
This comment didn't help me much.  Maybe it could say that "link" is something like "URL"?